android/app/src/main/assets/event

```diff
@@ -1,109 +1,52 @@
 {
-    "event_url": "http://2017.fossasia.org",
-    "description": "<p>The FOSSASIA Summit 2017 is an exciting Open Technology event for developers, students, and start ups using open source to build up their businesses. It is a unique opportunity to share the latest innovations in open source technologies and meet contributors and potential partners. It will be held at the Singapore Science Center in March 2017.</p><p>Since the first FOSSASIA meetup in 2009 in Ho Chi Minh City, FOSSASIA has taken the lead to organize Free and Open Source Software conferences in Asia. Past FOSSASIA events have attracted thousands of participants, speakers from 35 countries, and hundreds of volunteers. The FOSSASIA Summithas an over the average participation of female technologists.</p>",
-    "licence_details": {
-        "name": "Attribution",
-        "compact_logo": "https://licensebuttons.net/l/by/3.0/80x15.png",
-        "url": "https://creativecommons.org/licenses/by/4.0",
-        "long_name": "Creative Commons Attribution 4.0 International License",
-        "logo": "https://licensebuttons.net/l/by/3.0/88x31.png",
-        "description": "This license lets others distribute, remix, tweak, and build upon the work, even commercially, as long as they credit the copyright holder for the original creation."
-    },
-    "start_time": "2017-03-17T09:00:00+08:00",
-    "ticket_url": "https://eventyay.com/e/45da88b7/",
-    "topic": "Science & Technology",
-    "thumbnail": "https://storage.googleapis.com/eventyay.com/events/6/thumbnail/R0IzYXZ6bT/1481017202.91.jpg",
+    "identifier": "e4a55e79",
+    "name": "FOSSASIA 2016",
+    "latitude": 1.346,
+    "longitude": 103.686,
+    "location-name": "15 Science Centre Rd, Singapore 609081",
+    "starts-at": "2017-06-05T14:42:19+05:00",
```

Comment:
@arp95 Please update file. Just attribute names needs to be changed. Changing values will break sample. 
Here event date is changing so it will not show any sessions in Schedule Fragment. Also add social links which you have removed.